sdk/search/azure-search/src/main/java/com/azure/search/models/IndexingResult.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+/**
+ * Status of an indexing operation for a single document.
+ */
+@Fluent
+public final class IndexingResult implements Serializable {
```

Comment:
There is this scary note in the backend implementation of this model that says "DO NOT USE THIS IN ANY INDEXER RELATED PUBLIC API MODELS".  Not sure who put it there but makes me think maybe we shouldn't include this model?  Is it referenced anywhere?

Reply:
@Careyjmac This is part of the response for the push Index API. It definitely needs to be here.

That note is a reminder to not re-use models from the Index API in the implementation of indexers. :)